documentation/TechnicalDesignDocument.tex

```diff
@@ -134,19 +118,41 @@
 
     %------------------------------------------------------------------------------------------------------------------------------------------------------- Design Choices
     \section{Design Choices}
-    Design decisions: discuss the motivation (arguments/reasons), consequences,
-    and alternatives for the decisions.
-    Onderverdeeld in epics ipv user stories, makkelijker te snappen, overzichtelijker, etc. beschrijven..
-
-    Description of custom protocols, data formats etc.\\
-    Security measures and considerations.\\
-    Coding conventions may be added (as appendix).\\
-    Diagrams need textual explanation of context, interpretation and underlying design decisions and reasoning.\\
-
+    \label{sec:designchoices}
     It has been decided to resizes texture assets to dimensions in the power of two.
     This is to maintain backwards compatibility for OpenGL ES 1.0, to keep all functionality (some libGDX features are otherwise not supported) and it is a bit more memory efficient.\cite{libgdxpottex}
 
+    \subsection[Communication MVA-MEA]{Receiving command or information from MVA and MEA}
+    \label{subsec:comchoice}
+    Concurrency is essential to this task.
+    This way the user interface and the marama logic are separated and would not delay each other.
```

Comment:
Are separated
Would not
Fix tenses!

Reply:
Probably the best to simply say that they 'Do not delay' each other.

Reply:
Are you going to make the changes, or should I?
Agreed that 'do' is better than 'would' in this case.


>Are separated
>Would not
>Fix tenses!

Why is every first letter capitalized? Is that a mistake or do I fail to see the reason behind it?

Reply:
You should feel free to.

And... No rEaL ReAsoN